presto-orc/src/main/java/com/facebook/presto/orc/reader/FloatSelectiveStreamReader.java

```diff
@@ -250,6 +250,32 @@ else if (nullsAllowed) {
     private int readNoFilter(int[] positions, int positionCount)
             throws IOException
     {
+        if (positions[positionCount - 1] == positionCount - 1) {
+            // no skipping
+            if (presentStream != null) {
+                // some nulls
+                int nullCount = presentStream.getUnsetBits(positionCount, nulls);
+                if (nullCount == positionCount) {
+                    allNulls = true;
+                }
+                else {
+                    for (int i = 0; i < positionCount; i++) {
+                        if (!nulls[i]) {
+                            values[i] = floatToRawIntBits(dataStream.next());
```

Comment:
You may want to set `value[i] = 0;` if it's null, since we enforced the check that null elements should be 0.  Btw. I think it's debatable if null elements should be 0 but we added this check some time ago.

Reply:
@yingsu00 I believe you refer to this PR https://github.com/prestodb/presto/pull/13852 It added logic to populate zeros for null integer values, but not for floats. Hence, I think this can be left as is.